src/components/NewStack.js

```diff
@@ -132,10 +132,16 @@ class NewStack extends PureComponent {
 
   onChange (idx, e) {
     const target = e.currentTarget
-    const value = target.value
+    let value = target.value
     const name = target.name
     let operation = this.state.operations[idx]
 
+    const operationDefinition = this.props.operations[operation.name]
+
+    if (operationDefinition && operationDefinition.properties[name].type === 'bool') {
```

Comment:
FYI I want to make this more generic in the future when we introduce better validation (and hence better transformation) of the passed options.